src/content/docs/apis/nerdgraph/examples/browser-monitoring-config-nerdgraph.mdx

```diff
@@ -0,0 +1,114 @@
+---
+title: "NerdGraph tutorial: Browser agent monitoring examples"
+tags:
+  - APIs
+  - NerdGraph
+  - Examples
+  - Browser
+metaDescription: Examples of using New Relic NerdGraph API to adjust a browser agent
+redirects:
+  - /docs/apis/nerdgraph/examples/export-dashboards-pdfpng-api-tutorial
+  - /docs/apis/nerdgraph/examples
```

Comment:
I don't understand this portion of the front matter

Reply:
This doc used to be under these links, but as it's been iterated the address has changed. This ensures that if people (or another doc) is still using these old URLs, they're still taken to the right place.

Reply:
Since this is new content and never had an address before, this part of the frontmatter can be left empty!